scaffolder-templates/build/argocd/pipelinerun.template

```diff
@@ -43,6 +45,12 @@ spec:
     - name: docker-credentials
       secret:
         secretName: docker-credentials
+{% if values.gitHost == "github.com" %}
```

Comment:
I would have reversed the condition as GH is the default in my mind, but this is OK too

Reply:
the problem is that there could be many gitlab clients, and It was not easy implementing substring / regex for finding "gitlab" in the host name. github is always github.com so the condition is easy.

Reply:
ah, ok, anyway as I said, I'm fine with it :)